File: GhostTourJPA/src/main/java/com/skilldistillery/ghosttour/entities/Comment.java
package com.skilldistillery.ghosttour.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

@Entity
public class Comment {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;

	private String name;
	
	private String comment;
	
	private String image;
	
	@ManyToOne
	@JoinColumn(name="review_id")
	private Review review;

	public Comment() {
		super();
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getComment() {
		return comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

	public String getImage() {
		return image;
	}

	public void setImage(String image) {
		this.image = image;
	}

	public Review getReview() {
		return review;
	}

	public void setReview(Review review) {
		this.review = review;
	}

	@Override
	public String toString() {
		return "Comment [id=" + id + ", name=" + name + ", comment=" + comment + ", image=" + image + "]";
	}
	
	
	
	

}

File: ngGhostTour/src/app/models/comment.ts
import { Review } from "./review";

export class Comment {

  id: number;
  name: string;
  comment: string;
  reviewId: number;
  image: string;



  constructor(
    id: number = 0,
    name: string = "",
    comment: string = "",
    reviewId: number = 0,
    image: string = ""


  ){
    this.id = id;
    this.name = name;
    this.comment = comment;
    this.reviewId = reviewId;
    this.image = image;


  }
}
